vscode-extension/src/editor_server/editorServer.ts

```diff
@@ -73,6 +74,8 @@ export class EditorServer {
       const modelRegistryPathArgs = modelRegistryPath
         ? ["--parsers-module-path", modelRegistryPath]
         : [];
+      const filePath = config.get(ENV_FILE_PATH) as string;
```

Comment:
oh, one last thing - should this be `as string | undefined`?

Reply:
same as below